Cargo.toml

```diff
@@ -36,9 +36,7 @@ critical-section = { version = "1.1.2" }
 defmt = { version = "0.3.5", optional = true }
 embassy-sync = { version = "0.6.0", optional = true }
 embassy-futures = { version = "0.1.1", optional = true }
-embassy-time-driver = { version = "0.1.0", features = [
-    "tick-hz-1_000_000",
-], optional = true }
+embassy-time-driver = { version = "0.1.0", optional = true }
 embassy-time = { version = "0.3.0", features = [
     "tick-hz-1_000_000",
```

Comment:
I think that the tick feature should also be removed for `emabssy-time`.
And we might need to add the feature in the examples, as done in embassy.

Reply:
Thanks for the comment. I'm back from holidays.

I'll get this merged soon.